pkg/scheduler/objects/allocation.go

```diff
@@ -92,7 +92,7 @@ func NewAllocation(uuid, nodeID string, ask *AllocationAsk) *Allocation {
 		bindTime:          time.Now(),
 		nodeID:            nodeID,
 		partitionName:     common.GetPartitionNameWithoutClusterID(ask.GetPartitionName()),
-		uuid:              uuid,
+		uuid:              ask.allocationKey + "-" + strconv.Itoa(ask.completedPendingAsk()),
```

Comment:
I would propose to rename in a follow up change:
* replace uuid with `allocationID` or `allocID`
* replace `GetUUID` with `GetAllocationID`

Reply:
As a preparation add a new `GetAllocationID` function with the same functionality as the `GetUUID` function in this change. Start calling that instead of introducing 60+ new calls to `GetUUID` to simplify and reduce churn in the follow up.